### ISSUE TITLE
getDefiningElement NullPointerException

### ISSUE THREAD

**Issue by maucoder-ingeint** [wt-io-it/odoo-pycharm-plugin#98]
Plugin Version: 0.6.7
IntelliJ Version: PyCharm 2022.2.3
IntelliJ Build: PC-222.4345.23

<details><summary>Unhandled exception in [StandaloneCoroutine{Cancelling}@180a6b09, EDT]</summary>

```
java.lang.NullPointerException: Cannot invoke "java.util.function.Supplier.get()" because "exceptionSupplier" is null
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getDefiningElement(OdooModelImpl.java:92)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.lambda$getDefiningElement$4(OdooModelImpl.java:63)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:928)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelImpl.getDefiningElement(OdooModelImpl.java:47)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooModel(OdooGoToDeclarationHandler.java:147)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooModel(OdooGoToDeclarationHandler.java:132)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTarget(OdooGoToDeclarationHandler.java:45)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationHandlerBase.getGotoDeclarationTargets(GotoDeclarationHandlerBase.java:14)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTargets(OdooGoToDeclarationHandler.java:33)
	at com.intellij.codeInsight.navigation.impl.GtdProvidersKt.fromGTDProvidersInner(gtdProviders.kt:30)
	...
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$4.invoke(InternalReadAction.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:264)
	at com.intellij.openapi.application.rw.InternalReadAction.runReadAction(InternalReadAction.kt:32)
	...
	at com.intellij.codeInsight.navigation.CtrlMouseHandler2$compute$2.invokeSuspend(CtrlMouseHandler.kt:221)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	...
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```
</details>

**Comment by bigbear3001** [comment]
looks the same as #43

**Comment by bigbear3001** [comment]
this has been fixed in 0.6.9 (pending JetBrains approval). if you are eager to test the new version, you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.9

**Comment by ap-wtioit** [comment]
0.6.9 has been approved and is available in the marketplace
